code/modules/mod/mod_theme.dm

```diff
@@ -855,8 +835,8 @@
 	armor = list(MELEE = 25, BULLET = 20, LASER = 20, ENERGY = 5, BOMB = 25, RAD = 0, FIRE = 150, ACID = 150) //Equivalent armor to Security MODsuits
 
 /datum/mod_theme/cosmohonk
-	name = "cosmohonk"
-	desc = "A suit by Honk Ltd. Protects against low humor environments. Most of the tech went to lower the power cost."
+	name = "'Cosmohonk' entertainer"
+	desc = "A highly specialized modsuit for use by clowns across the Sector. Excellent power efficiency, godawful everything else."
```

Comment:
Losing the low-humor environments bit is a little tragic

Reply:
low-humor environment is present in the extended description, its a question of whether we want to repeat this joke in both places.